dev-docs/bidders/viewdeosDX.md

```diff
@@ -1,11 +1,13 @@
 ---
 layout: bidder
-title: ViewDeos DX
+title: Viewdeos
 description: Prebid ViewDeos Bidder Adaptor
 pbjs: true
+pbs: true
 biddercode: viewdeosDX
 media_types: banner,video
 gdpr_supported: true
+gvl_id: 924
 ---
 
```

Comment:
I see a few other parameters like `placementId`, `siteId` and `bidFloor` as well in your Prebid-server PR: https://github.com/prebid/prebid-server/pull/1846. Shouldn't those be included in this doc too?